### DISCUSSION TITLE
refind conf

### DISCUSSION THREAD

**Discussion by eqfae** [zbm-dev/zfsbootmenu#651] [Questions]
### ZFSBootMenu build source

Release EFI

### ZFSBootMenu version

2.3.0

### Problem description

```
timeout 2
shutdown_after_timeout false
resolution max
enable_mouse true
mouse_speed 22

"Boot default"  "zbm.prefer=r ro quiet loglevel=0 zbm.skip"
"Boot to menu"  "zbm.prefer=r ro quiet loglevel=0 zbm.show"
"Boot 11"  "root=ZFS=r/ROOT/linux ro quiet loglevel=0 zbm.skip" spl_hostid=00000000"

menuentry "go" {
    icon /EFI/refind/icons/os_linux.png
    loader /EFI/ZBM/zfsbootmenu.efi
    options "quiet loglevel=0 zbm.skip"
    disabled
}
```
At the end of /efi/EFI/refind/refind.conf, none of the configurations for the boot items take effect.

Minor modifications based on documentation
```
cat << EOF > /efi/EFI/ZBM/refind_linux.conf
"Boot default"  "quiet loglevel=0 zbm.skip"
"Boot to menu"  "quiet loglevel=0 zbm.show"
EOF
```
It also didn't help that modifying /boot/refind_linux.conf also

What the refind-install command does is create a default configuration, if there are no files in /boot/refind_linux.conf and /efi/EFI/refind/refind.conf.
It's the latter that takes effect, and I've tested it with different commands.
What I want is an option to go directly into the system without having to wait 10s, just wait 2s at the refind, and then go into the default if there's an exception.

Default generated configuration /boot/refind_linux.conf
```
"Boot with standard options"  "root=ZFS=r/ROOT/linux quiet spl_hostid=00000000"
"Boot to single-user mode"    "root=ZFS=r/ROOT/linux quiet spl_hostid=00000000 s
"Boot with minimal options"   "ro root="
```

### Steps to reproduce

I configured it mostly based on zfsbootmemu There should only be minor differences such as efi partitioning

**Comment by samy-mahmoudi** [comment]
Hi @eqfae,

It looks like you wrote menu entry `go` out of a sample of `refind.conf` but forgot to get rid of directive `disabled`:
```
#  disabled  - use alone or set to "yes" to disable this entry.
```

**Comment by samy-mahmoudi** [comment]
Besides, with a ZFSBootMenu self-contained UEFI application I suggest you to avoid `refind_linux.conf`.

An entry at the end of your `/efi/EFI/refind/refind.conf` should be sufficient.
